sysroot/usr/include/kernel/lib/LinkedList.hpp

```diff
@@ -59,7 +59,7 @@ class LinkedListNode {
      *
      * @return T Stored data
      */
-    T GetData()
+    T Data()
```

Comment:
Maybe make this return a `T&` for more efficient mutation (unless we want the data to immutable)

Reply:
Resolved in 9e01cca.